waspc/examples/todoApp/src/client/Todo.tsx

```diff
@@ -165,10 +171,12 @@ const NewTaskForm = () => {
       >,
     ],
   })
+  const createNewTask2 = tasksCrud.create.useAction()
 
   const createNewTask = async (description: Task['description']) => {
     const task = { isDone: false, description }
-    await createTaskFn(task)
+    // TODO: fails becasue it's not connected to auth entity
```

Comment:
For our example app, `tasksCrud.create.useAction()` is not a drop-in replacement since in the "manual" implementation there a connection to the `user`. 

We could detect that the `Task` has `user` as one of the fields and ... automatically add that connection. Sounds interesting to me, covers more use-cases than if we don't do it, but is that going too far with assumptions?

Reply:
What was the conclusion here, do we have any support for these kind of connections, especially with the user?

Reply:
This would fall under the "smart defaults" and we are not doing that in this iteration (as per our agreement if I'm not mistaken?)